.github/actions/vtk-install-dep/action.yml

```diff
@@ -17,6 +17,10 @@ inputs:
     description: 'Label to control egl'
     required: false
     default: 'no-egl'
+  cpu:
+    description: 'Label to control cpu architecture'
```

Comment:
Not a `label` (its a concept I created, not someting from github). description could be:

`CPU architecture to build for`